_layouts/docs.html

```diff
@@ -15,16 +15,23 @@
       <nav class="p-sidebar-nav">
         <ul class="p-sidebar-nav__list" id="docs-list-unsorted">
           <li class="p-sidebar-nav__item">
-            <strong>Docs</strong>
+            <h4>Documentation</h4>
+            <hr>
             <ul class="p-sidebar-nav__list" id="docs-list-unsorted">
-              <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/">User Guide</a></li>
+              <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/">Get started</a></li>
               <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/ports">Services and ports used</a></li>
               <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/addons">Addons</a></li>
-              <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/working">Working with registries</a></li>
               <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/clustering">Multi-node cluster</a></li>
               <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/release-channels">Release channels
                   and upgrades</a></li>
             </ul>
+            <strong>Images and registries</strong>
+            <ul class="p-sidebar-nav__list" id="docs-list-unsorted">
+               <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/registry-images">Using local images</a></li>
+               <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/registry-public">Using a public registry</a></li>
+               <li class="p-sidebar-nav__item"><a class="p-link--soft" href="/docs/registry-temporary">Using the temporary private registry</a></li>
```

Comment:
I do not like the "temporary" much. It would be better if we said "add-on registry" or "build-in registry"?

Reply:
that was how @tvansteenburgh wrote it in the overview doc. I don't have strong feelings on it

Reply:
> that was how @tvansteenburgh wrote it in the overview doc

I think it was me writing this overview doc